server/game/cards/03_TWI/upgrades/HoldoutBlaster.ts

```diff
@@ -0,0 +1,32 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { Card } from '../../../core/card/Card';
+import { UpgradeCard } from '../../../core/card/UpgradeCard';
+import { Trait, WildcardCardType, ZoneName } from '../../../core/Constants';
+
+export default class HoldoutBlaster extends UpgradeCard {
+    protected override getImplementationId () {
+        return {
+            id: '7280804443',
+            internalName: 'holdout-blaster',
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.setAttachCondition((card: Card) => !card.hasSomeTrait(Trait.Vehicle));
+
+        this.addWhenPlayedAbility({
+            title: 'You may have attached unit deal 1 damage to a ground unit.',
+            optional: true,
+            immediateEffect: AbilityHelper.immediateEffects.selectCard({
```

Comment:
Sorry, missed this on the first pass - by default we should use `targetResolver` instead of `selectCard()`. The latter has less functionality and is less easy to read. It's really just intended to be used in cases where we can't have `targetResolver`, like inside of a `sequential` or `conditional`. At some point we plan to migrate to find a way to only use `targetResolver` everywhere.